runtimes/common/src/constants.rs

```diff
@@ -194,11 +204,11 @@ pub mod governance {
 }
 
 pub mod did {
-	use crate::BlockNumber;
+	use super::*;
 
-	use super::{Balance, HOURS, KILT, MILLI_KILT};
+	pub const DID_SIZE: u32 = 7418;
```

Comment:
A comment explaining the result would be helpful, so that we can update this value if we make changes to the DID pallet.